RGPosadas/LitterRally
# TASK-82: Disallow Users Joining Past Events

This PR closes #283

Prerequisites: 
1. Make sure you have an event present in the database you're using that has a end date that is before the current date. 

To test old event not appearing anymore: 

1. Run Mull-Ui and Mull-Api
2. Navigate to discover/upcoming/my events
3. Check if there are any events with the end date that are before the current date appearing on the list

To test buttons not appearing on past event cards:
1. Remove this line in `event.service.ts`
![image](https://user-images.githubusercontent.com/35787279/112092571-28141400-8b6e-11eb-8a5e-b7f775717b16.png)
2. Run Mull-Ui and Mull-Api
3. Check that old events are now appearing on the discover page
4. Click an old event and check if the join buttons are not present on the `event card` and `preview page`
